build.py

```diff
@@ -1075,6 +1091,33 @@ def dockerfile_prepare_container_linux(argmap, backends, enable_gpu,
  && rm -f ${_CUDA_COMPAT_PATH}/lib
 '''
 
+    elif 'pytorch' in backends:
+        # The cpu-only build uses ubuntu as the base image, and so the cuda,
+        # openmpi, nccl and cudnn libs are not available. We must copy these
+        # pytorch dependencies from the Triton min container ourselves.
```

Comment:
Also add a line about how the pytorch binary is not CPU only hence have cuda deps.

Reply:
Already added in L1267 inside container_build()
```
    # For cpu-only image we need to copy some cuda libraries and dependencies
    # since we are using a PyTorch container that is not CPU-only
```

Reply:
Yes. Can you add a line here because we are copying the library here.

Reply:
Are you suggest to _replicate_ these lines to the section under L1094 or _move_ those lines to this section?

Reply:
I think you might have had some reason to describe the situation in L1267. So, I am just saying that we must be more descriptive here and justify why we are copying cuda files when enable_gpu is false. Something like below helps:
```
Add dependencies for pytorch backend. Note: Even if the build is cpu-only, pytorch depends upon libraries like cuda and cudnn, which are not present in ubuntu base image. Hence must copy these pytorch dependencies from the Triton min container ourselves.
```

Reply:
Done